src/scripts/settings/configMenu.lua

```diff
@@ -0,0 +1,292 @@
+-- Einstellungsdialog
+
+local menuData = nil
+
+function createConfigMenu(caption, data)
+    return {
+        caption = caption,
+        lines = data
+    }
+end
+
+function testMenu()
+    local menu = createConfigMenu("Automapper", {
+        {
+            lineNumber = -1,
+            configID = "mapper.roomDistance",
+            caption = "Abstand zwischen den Raeumen",
+            desc = "Langer Text hier als Beschreibung\nHoffentlich auch mehrzeilig...",
```

Comment:
Vermutlich wäre sinnvoll, auch diese Infos also Name und Beschreibung der Option in der Datenbank zu speichern, so dass sie in den Skripten zentral definiert werden können. Dazu ggf. setConfig() erweitern, oder neue Funktion ergänzen?